library/core/src/clone.rs

```diff
@@ -51,7 +47,7 @@
 /// ## Derivable
 ///
 /// This trait can be used with `#[derive]` if all fields are `Clone`. The `derive`d
-/// implementation of [`clone`] calls [`clone`] on each field.
+/// implementation of [`Clone::clone`] calls [`Clone::clone`] on each field.
```

Comment:
Same comment, the original text is less repetitive.
```suggestion
/// implementation of [`Clone`] calls [`clone`] on each field.
```